wagtail/images/tests/test_image_operations.py

```diff
@@ -616,6 +616,30 @@ def test_run_raises_error(self):
 
 
 class TestFormatFilter(TestCase):
+    def test_avif(self):
```

Comment:
Please extend the `test_image_operations` with a `TestAvifQualityFilter` test case, similar to [`TestJPEGQualityFilter`](https://github.com/wagtail/wagtail/blob/32ff36446b0913c3180d23848552287726a228cc/wagtail/images/tests/test_image_operations.py#L682) or [`TestWebPQualityFilter`](https://github.com/wagtail/wagtail/blob/32ff36446b0913c3180d23848552287726a228cc/wagtail/images/tests/test_image_operations.py#L762)